app/editor/src/features/content/morning-report/constants/defaultReportFilter.ts

```diff
@@ -12,7 +12,7 @@ export const defaultReportFilter: IMorningReportFilter = {
   sourceId: 0,
   otherSource: '',
   productIds: [],
-  sourceIds: [],
+  sourceIds: [91, 248, 249, 246, 247],
```

Comment:
This is always problematic, as there is no guarantee that these IDs mean the same thing in each environment.  Do it by code instead.  Code isn't a ton better, but at least it's human readable and manageable through the interface.